demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1112,6 +1132,8 @@ export class DemoMeetingApp
 
   metricsDidReceive(clientMetricReport: ClientMetricReport): void {
     const metricReport = clientMetricReport.getObservableMetrics();
+    this.videoUpstreamMetricReport = clientMetricReport.getObservableVideoUpstreamMetrics();
+    this.videoDownstreamMetricReport = clientMetricReport.getObservableVideoDownstreamMetrics();
     if (
```

Comment:
Dont we want to show the audio stats here?

Reply:
So far we don't have audio metrics showing in the meeting demo. I will add them in the component library demo app.